SwiftWisdom/Core/StandardLibrary/NSAttribitedString/NSMutableAttributedString+Format.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  NSAttributedString+Format.swift
+//  bose-connect-ios
+//
+//  Created by Paul Rolfe on 1/16/16.
+//  Copyright © 2016 Intrepid-Pursuits. All rights reserved.
+//
+
+import Foundation
+
+extension NSMutableAttributedString {
+    
+    /**
+     Create a mutable attributed string with the attributes applied to all arguments.
+     
+     - parameter string:     formatted string where `%@` represents an argument.
+     - parameter attributes: attributes dictionary to use on arguments
+     - parameter args:       comma separated list of arguments
+     
+     - returns: NSMutableAttributedString with the attributes applied to argument strings.
+     */
+    public convenience init(stringWithFormat string: String, applyingAttributes attributes: [String : AnyObject], toArgs args: String...) {
+        let formattedArgs = args.map { arg in
+            return NSAttributedString(string: arg, attributes: attributes)
+        }
+        
+        self.init(stringWithFormat: string, withArguments: formattedArgs)
+    }
+    
+    /**
+     Create a mutable attributed string with the given list of NSAttributedString
+     
+     - parameter string: formatted string where `%@` represents an argument.
+     - parameter args:   comma separated list of arguments (NSAttributedString)
+     
+     - returns: NSMutableAttributedString with arguments inserted according to format.
+     */
+     public convenience init(stringWithFormat string: String, _ args: NSAttributedString...) {
+        self.init(stringWithFormat: string, withArguments: args)
+    }
+    
+    /**
+     Create a mutable attributed string with the given collection of NSAttributedString
+     
+     - parameter string: formatted string where `%@` represents an argument.
+     - parameter args:   comma separated list of arguments (NSAttributedString)
+     
+     - returns: NSMutableAttributedString with arguments inserted according to format.
+     */
+    private convenience init(stringWithFormat string: String, withArguments args: [NSAttributedString]) {
```

Comment:
any reason to have this `private`? Might be useful to be able to call w/ an array as well.  I might change the above `_ args` to match this signature so the overload is clear that they can use either.  

Overloading w/ array seems to be convention until splatting is explicitly supported